test/parallel/test-vm-context-dont-contextify.js

```diff
@@ -0,0 +1,185 @@
+'use strict';
+
+// Check vm.constants.DONT_CONTEXTIFY works.
+
+const common = require('../common');
+
+const assert = require('assert');
+const vm = require('vm');
+const fixtures = require('../common/fixtures');
+
+{
+  // Check identity of the returned object.
+  const context = new vm.createContext(vm.constants.DONT_CONTEXTIFY);
```

Comment:
Is the `new` necessary?

```suggestion
  const context = vm.createContext(vm.constants.DONT_CONTEXTIFY);
```